hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala

```diff
@@ -137,6 +137,9 @@ object HoodieSparkSqlWriter {
       val partitionColumns = HoodieSparkUtils.getPartitionColumns(keyGenerator, toProperties(parameters))
       // Create the table if not present
       if (!tableExists) {
+        val preCombineField = hoodieConfig.getString(PRECOMBINE_FIELD)
```

Comment:
will remove this line. just a note to reviewer that here the value is already deduced to "ts" if not explicitly set by the user.